docs/source/getting_started/installation.rst

```diff
@@ -170,6 +170,18 @@ To build vLLM using an existing PyTorch installation:
     $ pip install -e . --no-build-isolation
 
 
+Use the local cutlass for compilation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Currently, before starting the build process, vLLM fetches cutlass code from GitHub. However, there may be scenarios where you want to use a local version of cutlass instead.
+To achieve this, you can set the environment variable CUTLASS_SRC_DIR to point to your local cutlass directory.
+
+.. code-block:: console
+
+    $ git clone https://github.com/vllm-project/vllm.git
+    $ cd vllm
+    $ CUTLASS_SRC_DIR=/path/to/cutlass pip install -e .
+
```

Comment:
I found that relative paths don't work for specifying the CUTLASS_SRC_DIR. 

This is a pretty easy thing for users to get tripped up on, so could you document that absolute paths must be specified (or get relative paths working if you see a way to do that)?

Reply:
Great idea, I've implemented a logic to convert relative paths to absolute paths directly in the CMake script. Please review it to see if it's feasible.